specification/network/resource-manager/Microsoft.Network/stable/2018-12-01/ddosCustomPolicy.json

```diff
@@ -0,0 +1,299 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "DDoSCustomPolicyClient",
+    "description": "The Microsoft Azure Network management API provides a RESTful set of web services that interact with Microsoft Azure Networks service to manage your network resources. The API has entities that capture the relationship between an end user and the Microsoft Azure Networks service.",
+    "version": "2018-12-01"
```

Comment:
2018-11-01

Reply:
done

Reply:
I did make the versions 2018-11-01. but please double check because every other json in the December branch seems to be using 2018-12-01.